src/app/Shopping/Detail/page.tsx

```
@@ -5,14 +5,31 @@ import { Tab, ProductCardSmall, Tag } from "../../../components/Shopping";
+
+const data = [
+  {
+    id: 1,
+    image: GlassDummy
+  }
+]
 
 const ShoppingDetail = () => {
   const [selectedTab, setSelectedTab] = useState<number>(1);
   const [selectedHeart, setSelectedHeart] = useState<boolean>(false);
+  const [selectedColor, setSelectedColor] = useState<number>(1);
```

ESLint

[error] 'setSelectedColor' is assigned a value but never used. Allowed unused elements of array destructuring patterns must match /^_/u.

(@typescript-eslint/no-unused-vars)